File: swing/src/main/java/wannabe/swing/renderer/Pixel.java
package wannabe.swing.renderer;

import java.awt.Graphics;
import wannabe.swing.SwingProjected;

public class Pixel extends SwingRenderer {
  @Override public void draw(Graphics g, SwingProjected p) {
    g.drawLine(p.left, p.top, p.left, p.top);
  }
}

File: swing/src/main/java/wannabe/swing/renderer/Pixel.kt
package wannabe.swing.renderer

import wannabe.swing.SwingProjected
import java.awt.Graphics

class Pixel : SwingRenderer() {
  override fun draw(
    g: Graphics,
    p: SwingProjected
  ) {
    g.drawLine(p.left, p.top, p.left, p.top)
  }
}
